fe2-android/app/src/test/java/com/github/dedis/popstellar/repository/remote/ConnectionTest.java

```diff
@@ -0,0 +1,74 @@
+package com.github.dedis.popstellar.repository.remote;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+import com.github.dedis.popstellar.model.network.method.Message;
+import com.github.dedis.popstellar.model.network.method.Subscribe;
+import com.tinder.scarlet.Lifecycle;
+import com.tinder.scarlet.ShutdownReason;
+
+import org.junit.Test;
+
+import io.reactivex.subjects.BehaviorSubject;
+
+public class ConnectionTest {
```

Comment:
In general `IsCorrect` isn't necessarily the best description for a test.
You want to aim for something descriptive, what is the precise, correct behaviour you're testing for? 
`Correct` might mean different things depending on what you're testing.

e.g. `sendMessageCallsDelegate`
`observeMessageCallsDelegate`
`connectionClosesGracefully`

Reply:
Done